curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/escaping-literal-quotes-in-strings.md

```diff
@@ -35,7 +35,7 @@ assert(code.match(/\\"/g).length === 4 && code.match(/[^\\]"/g).length === 2);
 Variable myStr should contain the string: `I am a "double quoted" string inside "double quotes".`
 
 ```js
-assert(myStr === 'I am a "double quoted" string inside "double quotes".');
+assert(/I am a "double quoted" string inside "double quotes.?".?/.test(myStr));
```

Comment:
Your regex would allow the following:
```js
I am a "double quoted" string inside "double quotes.".
```

```suggestion
assert(/I am a "double quoted" string inside "double quotes(\."|"\.)$/.test(myStr));
```